pkg/sdn/plugin/node.go

```diff
@@ -252,7 +256,9 @@ func (node *OsdnNode) Start() error {
 	if err = node.policy.Start(node); err != nil {
 		return err
 	}
-	node.watchServices()
+	if !node.useConnTrack {
+		go kwait.Forever(node.watchServices, 0)
+	}
```

Comment:
Doh. A bad rebase after the shared informers PR ended up turning the node process into a fork bomb when not using networkpolicy.